src/pages/loading-waiting-states/contextual-spinner/ContextualSpinner.tsx

```diff
@@ -0,0 +1,123 @@
+import React, { useState } from 'react';
+import { makeStyles, useTheme, Theme } from '@material-ui/core/styles';
+import { AppBar, Toolbar, Hidden, IconButton, Typography, Button, Fab, CircularProgress } from '@material-ui/core';
+import { TOGGLE_DRAWER } from '../../../redux/actions';
+import { useDispatch } from 'react-redux';
+import { Menu, PlayArrow } from '@material-ui/icons';
+
+const useStyles = makeStyles((theme: Theme) => ({
+    root: {
+        backgroundColor: theme.palette.background.paper,
+        minHeight: '100vh',
+    },
+    appbarRoot: {
```

Comment:
Based on the designs this should probably be omitted anyway. All the old examples use this, but we should probably be working on removing it from the library entirely rather than building a component for it.